### PR TITLE
Initial project

### PR DESCRIPTION
**Pull request by gigawave07**
Please review my first page. Thank you.

### CODE REVIEW

File: 1st_page.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```
Comment (Minhnguyendhth8b) [inline 1st_page.html]: Rename this file to index.html

**Review by Minhnguyendhth8b — CHANGES_REQUESTED** [review]
Please rename file html to index.html and file css to styles.css

File: 1st_page.html (hunk 1 above)
Comment (gigawave07) [reply 1st_page.html]: done

**Comment by Minhnguyendhth8b** [comment]
- And file style.css to styles.css also

**Pull request merged**